CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 - Fix error raising while serializing JSON due NaN ([#112])
 
+### New Features
+
+- Add pydatic request models and documentation to API ([#102])
```

Comment:
If you have this change as a “new feature” in the changelog, I would maybe add an additional URL to the pydantic documentation or an explanation what pydantic is.

Reply:
Reformulated the senctence to include more information and added links to pydatic docs and fastAPI docs.